syntax = "proto3";

service FileService {
  rpc UploadFile (UploadFileRequest) returns (UploadFileResponse);
  rpc GetFile (GetFileRequest) returns (GetFileResponse);
}

message UploadFileRequest {
  string file_name = 1;
  bytes file_content = 2;
}

message UploadFileResponse {
  string message = 1;
}

message GetFileRequest {
  string file_name = 1;
}

message GetFileResponse {
  bytes file_content = 1;
}
